components/brave_wallet/browser/nonce_tracker.cc

```diff
@@ -0,0 +1,52 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/brave_wallet/browser/nonce_tracker.h"
+
+#include <algorithm>
+#include <utility>
+
+#include "base/bind.h"
+#include "brave/components/brave_wallet/browser/json_rpc_service.h"
+#include "brave/components/brave_wallet/browser/tx_meta.h"
+#include "brave/components/brave_wallet/browser/tx_state_manager.h"
+#include "brave/components/brave_wallet/common/brave_wallet.mojom.h"
+#include "brave/components/brave_wallet/common/fil_address.h"
+#include "third_party/abseil-cpp/absl/types/optional.h"
```

Comment:
From a quick look, bind.h, tx_meta.h, fil_address.h is not needed, please remove and double check if there are any other unnecessary headers.

Reply:
done